xgcm/autogenerate.py

```diff
@@ -1,8 +1,7 @@
-from __future__ import print_function
-from future.utils import iteritems
```

Comment:
Which hook removes the unused modules? That is awesome! Is that flake?

Reply:
`flake8` reported them, and I had to remove them manually.